hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1015,7 +1015,7 @@ public void testSnapshotStatsTableProperties() throws Exception {
   @Test
   public void testSetSnapshotSummary() throws Exception {
     Configuration conf = new Configuration();
-    conf.set("iceberg.hive.table-property-max-size", "4000");
+    final long maxHiveTablePropertySize = 4000;
     HiveTableOperations ops =
         new HiveTableOperations(conf, null, null, catalog.name(), DB_NAME, "tbl");
```

Comment:
Both `conf` and `ops` are no longer needed. Instead of calling methods on `ops`, you now call static methods in `HMSTablePropertyHelper`.

Reply:
Fixed.